prisma/schema.prisma

```diff
@@ -9,171 +8,277 @@ datasource db {
 
   modificationGroup ModificationGroup @relation(fields: [modificationGroupId], references: [id])
 
   @@unique([modificationGroupId, name])
 }
 
-model Order{
-  id String @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
-  customerId String @db.Uuid 
-  restaurantId String @db.Uuid
-  orderDate DateTime @default(now())
-  status String
-  totalAmount Int
-  freeDelivery Boolean    @default(false)
-  address String
-  createdAt DateTime @default(now())
-  updatedAt DateTime @default(now())
+model Order {
+  id           String   @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  customerId   String   @db.Uuid
+  restaurantId String   @db.Uuid
+  orderDate    DateTime @default(now())
+  status       String
+  totalAmount  Int
+  freeDelivery Boolean  @default(false)
+  address      String
+  createdAt    DateTime @default(now())
+  updatedAt    DateTime @default(now())
 
   orderLines OrderLine[]
 
   restaurant Restaurant @relation(fields: [id], references: [orderId])
-  customer Customer @relation(fields: [customerId],references: [id])
+  customer   Customer   @relation(fields: [customerId], references: [id])
 }
 
-model OrderLine{
-  id String @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
-  menuItemId String @db.Uuid
-  orderId String @db.Uuid 
-  quantity Int
+model OrderLine {
+  id         String   @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  menuItemId String   @db.Uuid
+  orderId    String   @db.Uuid
+  quantity   Int
   totalPrice Int
-  createdAt DateTime @default(now())
-  updatedAt DateTime @default(now())
-  
-  menuItem MenuItem @relation(fields: [menuItemId],references: [id])
-  order Order @relation(fields: [orderId],references: [id])
-}
\ No newline at end of file
+  createdAt  DateTime @default(now())
+  updatedAt  DateTime @default(now())
+
+  menuItem MenuItem @relation(fields: [menuItemId], references: [id])
+  order    Order    @relation(fields: [orderId], references: [id])
+}
+
+model Supplier {
+  id         String       @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  name       String
+  contacts   String
+  bin        String
+  Ingredient Ingredient[]
+}
+
+enum Unit {
+  LITRES
+  KG
+  G
+}
+
+model Ingredient {
+  id         String  @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  supplierId String  @db.Uuid
+  name       String
+  yield      Decimal @db.Decimal(4, 2)
+  unit       Unit    @default(G)
+
+  supplier                 Supplier                   @relation(fields: [supplierId], references: [id])
+  RecipeIngredient         RecipeIngredient[]
+  InventoryCountIngredient InventoryCountIngredient[]
+  StorageIngredient        StorageIngredient[]
+}
+
+model Recipe {
+  id                  String             @id @default(dbgenerated("uuid_generate_v4()")) @db.Uuid
+  name                String
+  unit                Unit               @default(G)
+  yield               Decimal            @db.Decimal(4, 2)
```

Comment:
I am not sure if this is correct. It can store only  values from -99.99 to 99.99 (4 - how much digits in total, 2 - how much digits in fraction part)

So, if you want to represent percentage from -100 to 100, you would need 5, 2

But the best practice often calls to store percentage as fraction. I would store them in decimal(5, 4). so you can store percentage like this 0.5932 (59.32%). The reason why I like this way is because it makes calculation easier, since I don't have to divide by 100 to do math. Also, the reason why I chose 4 precision, is because now I can store double digit fraction, which is 0.32% in this example